docs/appendices/e.rst

```diff
@@ -20,7 +20,7 @@ interesting common feature: FREE STUFF. Our list of SQL web sites includes:
 
 Our selection is primarily of English-speaking sites. As URLs change, and as
 new sites match our criteria, we will update a copy of this list at our own
-web site: http://ourworld.compuserve.com/homepages/OCELOTSQL/sites.htm
+web site: http\://ourworld.compuserve.com/homepages/OCELOTSQL/sites.htm (dead)
```

Comment:
@amotl Do you think it would be better to link to Internet Archive for this dead link?

https://web.archive.org/web/20021015234703/http://ourworld.compuserve.com/homepages/OCELOTSQL/sites.htm

Reply:
Sure, please use the link to the Internet Archive.